examples/StylingExample.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright Schrodinger, LLC
+ */
+
+"use strict";
+
+const FakeObjectDataListStore = require('./helpers/FakeObjectDataListStore');
+const { ImageCell, TextCell } = require('./helpers/cells');
+const { Table, Column, Cell } = require('fixed-data-table-2');
+const React = require('react');
+const {StyleSheet, css} = require('aphrodite');
+
+class StylingExample extends React.Component {
```

Comment:
Hopefully this example helps with issues like https://github.com/schrodinger/fixed-data-table-2/issues/51 and https://github.com/schrodinger/fixed-data-table-2/issues/47

I found some limitations though.  The 3 wraps on the default cell make styling it hard.  I couldn't style the border of FixedDataTableCellDefault.  Also when the table width > total column width, I couldn't figure out how to style the extra header padding.